src/main/resources/checkstyle/checkstyle-hubspot.xml

```diff
@@ -128,6 +128,10 @@
       <property name="ignoreEnhancedForColon" value="false"/>
     </module>
 
+    <module name="WhitespaceAfter">
+      <property name="tokens" value="COMMA, SEMI" />
```

Comment:
can you add TYPECAST as well, and also add an `id` property to the module (we try to give every module an ID so that users can disable checkstyle by module name if needed)